src/alitra/models/bounds.py

```diff
@@ -1,40 +1,40 @@
 from dataclasses import dataclass
 
-from alitra.frame_dataclasses import Point
+from .position import Position
 
 
-# Point 1 and 2 should be on the diagonal corners of the bounding cube
 @dataclass
 class Bounds:
-    point1: Point
-    point2: Point
+    """
+    Bounds defines the cube bounds of which the map should be valid inside.
+    Position 1 and 2 should be on the diagonal corners of the bounding cube.
+    """
+
+    position1: Position
+    position2: Position
 
     def __post_init__(self):
-        if self.point1.frame == self.point2.frame:
-            self.frame = self.point1.frame
+        if self.position1.frame == self.position2.frame:
+            self.frame = self.position1.frame
         else:
-            raise FrameException("The frames of the bounding points are not the same")
-        points: list = [self.point1, self.point2]
-        self.x_max = max(point.x for point in points)
-        self.x_min = min(point.x for point in points)
-        self.y_max = max(point.y for point in points)
-        self.y_min = min(point.y for point in points)
-        self.z_max = max(point.z for point in points)
-        self.z_min = min(point.z for point in points)
+            raise TypeError("The frames of the bounding Positions are not the same")
```

Comment:
```suggestion
            raise ValueError("The frames of the bounding Positions are not the same")
```
Wouldn't `ValueError` be more correct?

Reply:
Agreed, fixed